stack/Pulumi.perth-neuro.yaml

```diff
@@ -0,0 +1,11 @@
+encryptionsalt: v1:SPQXlVggjxw=:v1:X1sTpViNuyK+Wcom:3GANI8gZOKtk0gG7BklsyHeNU5uVLw==
+config:
+  datasets:customer_id: C010ys3gt
+  datasets:depends_on: '["seqr"]'
```

Comment:
This is backwards -- seqr depends on perth-neuro, not the other way around.

Reply:
Ahh, my initial reading was right.